crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -517,24 +532,81 @@ type NameServiceEdge {
 }
 
 type Object implements ObjectOwner {
+	"""
+	If this object is a MoveObject, this is a number that increases each time a tx takes this object as a mutable input
+	This is a lamport timestamp, not a sequentially increasing version
+	If this object is a MovePackage, the version is increased every time the package is upgraded
+	Note that packages are referred to by move calls using just their ID, and they are always loaded at their latest version
```

Comment:
I'm not sure I would go into this much detail on how versions work here -- we have a docs page for that.  If anything I would drop the explanation of this field altogether -- there's nothing to say other than thiat this is the version of the object.